index.html

```diff
@@ -0,0 +1,169 @@
+    margin: 0;
+    padding: 0;
+}
+ .a {
+    float: right;
+    display: block;
+    text-color:#000000;
+    padding: 8px;
+    text-decoration: none;
+    text-align: right;
+    text-transform: capitalize;
+        
+}
+
+main {
+        float: left;
+        display: block;
+        color: #000000;
+        padding: 8px;
+        text-decoration: none;
+        text-align: left;
+        text-transform: capitalize;
+}
+a:hover,
+main:hover
+ {
+    background-color: #c5c6c7;
+}
+
+h1 {
+color: darkorange;
+text-transform: capitalize;
+text-align: center;
+font-style: oblique;
+font-family: monospace;
+text-shadow: #000000;
+}
+
+h2 {
+color:gold;
+text-transform: uppercase;
+text-align: center;
+font-style: italic;
+font-family: serif;
+} 
+
+p {
+color:#ffffff;
+}
+
+.click {
+  background-color: green;
+  width: 100px;
+  border: 15px thick black;
+  padding: 20px;
+  margin: 10px;
+  text-align: center;
+  background-position: left middle;
+}
+img {
+    display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 30%;
+  border-radius: 50%;
+  opacity: 0.4;
+
+}
+section {
+
+} 
+article {
+text-align: center;
+list-style-type: square;
+font-style: italic;
+}
+ footer {
+   background-color: black;
+   color:tomato;
+   text-align: center;
+   text-indent: 2px;
+   font-weight: 700;
+   font-style: italic;
+
+}
+          </style>
+            <title>Bimbolabuari</title>
+            <nav>
+                <ol>
+                    <li class="a"><a href="aboutme" class="aboutme">ABOUT ME</a></li>
+                    <li class="a"><a href="projects" class="projects">PROJECTS</a></li>
+                    <li class="a"><a href="contacts" class="contacts">CONTACTS</a></li>
+                </ol>
+            </nav>
+            <main>
+              Abimbola Buhari<br>
+              A website designer/developer
+            </main>
+        </header>
+        <section>
+        <img src="abimbola.jpg" alt="my image"><br>
+        <p class="click">Click here to view more <a href="http://myphotoapp.com">personal photos</a></p>
+        </section>
+
+        <h1> Hey,Am Abimbola Buari</h1>
+
+        <h2>Do You Want To Know About Me?</h2>
+
+        <article> 
+          <h3>Education Status:</h3>
+          I have a b.sc in entrepreneurship
+        </article>
+
+        <article>
+          <h3>Hobbies:</h3>
+          <ul>
+          <li>Reading</li>
+          <li>Travelling</li>
+          <li>sleeping</li>
+          <li>Exercising</li>
+          </ul>
+        </article>
+
+        <article>
+          <h3>Skills:</h3>
+          <ul>
+            <li>Sewing</li>
+            <li>Make Over</li>
+            <li>Website developing</li>
+          </ul>
+        </article>
+      
+        <footer>
+          <p><a href="contacts">Contacts Info</a></p>
+            <p id="contacts-header">07065750214<br>08126028008</p>
+              Written by <a href="abimbola.buari@gmail.com">Abimbola Aminat</a>.<br> 
```

Comment:
Use anchor tags for both email and tel links.

Check https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a appropriate `href` value to be used.